crates/oxc_ast/src/ast/js.rs

```diff
@@ -1955,6 +1955,8 @@ pub enum ExportDefaultDeclarationKind<'a> {
 #[cfg_attr(feature = "serialize", derive(Serialize, Tsify))]
 #[cfg_attr(feature = "serialize", serde(untagged))]
 pub enum ModuleExportName<'a> {
-    Identifier(IdentifierName<'a>),
+    IdentifierName(IdentifierName<'a>),
```

Comment:
Where's the need for this?

Reply:
All other usages are still `IdentifierName`.

Reply:
I think we should just need `IdentifierReference`. But it may require a lot of changes.